content/php-client/resources/PAM/asset-variation-files.md

```diff
@@ -70,6 +77,13 @@ $product = $client->getAssetVariationFileApi()->downloadFromNotLocalizableAsset(
 file_put_contents('/tmp/bridge-mobile.jpg', $product->getContents());
 ```
 
+From v4 of the PHP client, the response is returned instead of the content. It allows to get the filename and the Mime type from the response.
```

Comment:
```suggestion
From the v4 of the PHP client, the response is returned instead of the content. It allows getting the filename and the MIME type from the response.
```